packages/server/graphql/queries/helpers/fetchJiraServerProjects.ts

```diff
@@ -0,0 +1,36 @@
+import {DataLoaderWorker} from './../../graphql'
```

Comment:
-1 I think this file is a merge artefact, the file should be deleted as it's now a dataloader.